### ISSUE TITLE
Lifecycle error

### ISSUE THREAD

**Issue by Sid-Turner-Ellis** [mancku/strapi-plugin-schemas-to-ts#33]
The types are generating fine for me but when the app starts it throws an error after the types are generated:

src/common/schemas-to-ts/BeforeRunEvent.ts:3:23 - error TS2307: Cannot find module '@strapi/database/lib/lifecycles/index' or its corresponding type declarations.

3 import { Event } from '@strapi/database/lib/lifecycles/index';
                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

This is occurring in the BeforeRunEvent.ts file that is created

**Comment by creazy231** [comment]
show https://github.com/mancku/strapi-plugin-schemas-to-ts/pull/29

**Comment by mancku** [comment]
Hi @Sid-Turner-Ellis @creazy231 I'm taking a look at https://github.com/mancku/strapi-plugin-schemas-to-ts/pull/29 to see if I can merge it right away or if something's gonna need to be done in order to keep the plugin compatible with older versions of Strapi

**Comment by mancku** [comment]
Hi @Sid-Turner-Ellis please, take a look at this comment to see if you can also test it: https://github.com/mancku/strapi-plugin-schemas-to-ts/pull/29#issuecomment-1802633255

**Comment by mancku** [comment]
Hi @Sid-Turner-Ellis I just released the version 1.1.20 of the plugin that fixes this. Thanks!
